libs/ui-components/src/components/Catalog/CatalogPage.css

```
@@ -1,3 +1,11 @@
 .fctl-catalog-page {
   padding-top: var(--pf-t--global--spacer--md);
 }
+
+.fctl-catalog-page__filters {
+  width: 250px;
+  min-width: 200px;
+  flex-shrink: 0;
+  word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)